src/qt/optionsmodel.cpp

```diff
@@ -54,6 +54,7 @@ static const char* SettingName(OptionsModel::OptionID option)
     case OptionsModel::ProxyPortTor: return "onion";
     case OptionsModel::ProxyUseTor: return "onion";
     case OptionsModel::Language: return "lang";
+    case OptionsModel::Privacy: return "privacy";
```

Comment:
In commit "Persist "mask values" in gui" (9fde0bae916f73b73fcdbeff6469dba72c4be212)

This change to `SettingName` function doesn't do anything and should be reverted. `SettingName` maps `OptionModel` enum values to `bitcoin.conf` / command line option names, and there is no "-privacy" option, so this wouldn't do the right thing even if the case was ever used.

If you want to use `OptionsModel` to read and write this setting, instead of changing the `SettingName` function, you should change the `OptionsModel::getOption()` function by adding:

```c++
    case Privacy:
        return settings.value("privacy");
```

And the `OptionsModel::setOption()` function by adding:

```c++
    case Privacy:
        settings.setValue("privacy", value.toBool());
        break;
```

You could grep `CoinControlFeatures` or `EnablePSBTControls` to see how another similar GUI settings implemented in `OptionsModel`.